04-onion-routing.md

```diff
@@ -1330,7 +1331,7 @@ A _forwarding node_ MAY, but a _final node_ MUST NOT:
     - report the `cltv_expiry` of the outgoing HTLC and the current channel setting for the outgoing
     channel.
     - return an `incorrect_cltv_expiry` error.
-  - if the `cltv_expiry` is unreasonably near the present:
+  - if the `cltv_expiry` is more than `max_htlc_cltv` near the present:
```

Comment:
I think this should be modifying the line below?

Reply:
Done at bccab9afc, thanks for the catch.